docs/ref/contrib/admin/index.txt

```diff
@@ -2635,6 +2635,12 @@ creating your own ``AdminSite`` instance (see below), and changing the
     See :ref:`customizing-adminsite` for an example of customizing the
     :class:`AdminSite` class.
 
+    By default, an instance of ``django.contrib.admin.sites.AdminSite``
+    is created as ``django.contrib.admin.site`` and you can
+    register your models and ``ModelAdmin`` instances with it.
+    If needed, it can be overridden with a custom one,
+    see :ref:`overriding-default-adminsite`.
```

Comment:
IMO the first sentence is redundant with existing documentation (see [docs](https://github.com/django/django/blob/master/docs/ref/contrib/admin/index.txt#L2625-L2628)). I think we should add only the second sentence, i.e.
```diff
diff --git a/docs/ref/contrib/admin/index.txt b/docs/ref/contrib/admin/index.txt
index c043561f17..ce93c842c4 100644
--- a/docs/ref/contrib/admin/index.txt
+++ b/docs/ref/contrib/admin/index.txt
@@ -2625,7 +2625,9 @@ creating your own ``AdminSite`` instance (see below), and changing the
     A Django administrative site is represented by an instance of
     ``django.contrib.admin.sites.AdminSite``; by default, an instance of
     this class is created as ``django.contrib.admin.site`` and you can
-    register your models and ``ModelAdmin`` instances with it.
+    register your models and ``ModelAdmin`` instances with it. If needed, it
+    can be overridden with a custom one, see
+    :ref:`overriding-default-adminsite`.
 
     When constructing an instance of an ``AdminSite``, you can provide
     a unique instance name using the ``name`` argument to the constructor. This
```

Reply:
Yes, it's nicer